### DISCUSSION TITLE
Update modules with script & CLI

### DISCUSSION THREAD

**Discussion by toonetcreation** [PrestaShop/PrestaShop#36620] [Q&A] [answered]
Hi,

I have is there a way to update prestashop modules (default + non-native modules/addons) without connecting manually on the backend?
For instance using a script from a remote server to upload the module and install/update it automatically ?

Thanks
L.

**Comment by matks** [comment, marked as answer]
Hello I think this is what you are looking for 😄 https://devdocs.prestashop-project.org/8/development/components/console/prestashop-module/

**Reply by toonetcreation** [reply]
many thanks for this feedback.
seems to be the right way yes.
do you know if this is working for prestashop 1.7.X releases?

**Comment by toonetcreation** [comment]
I had a look and seems to be onlu for prestashop 8.
